### PR TITLE
github action: CI job

### PR DESCRIPTION
**Pull request by richardm-stripe**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        fetch-depth: 0
+        submodules: recursive
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.16
+    - name: test
+      run: |
+        go generate &&
+        make
```
Comment (akalinin-stripe) [inline .github/workflows/ci.yml]: It looks like `make` without additional args, will run all the steps: `test vet lint check-gofmt build`

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      with:
+        fetch-depth: 0
+        submodules: recursive
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.16
+    - name: test
+      run: |
+        go generate &&
```
Comment (akalinin-stripe) [inline .github/workflows/ci.yml]: I don't know if go-bindata exists in the default build container, in goreleaser, it's installed as pre-req: https://github.com/stripe/stripe-mock/pull/266/files#diff-1c7213c2f02fce0c1d653f1fe32831b700931145e3d5b7245c0df732c492aba8R6

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        with:
+          fetch-depth: 0
+          submodules: recursive
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
+      - name: test
+        run: |
+          go install golang.org/x/lint/golint@latest &&
```
Comment (akalinin-stripe) [inline .github/workflows/ci.yml:21]: Since golint is deprecated, this should be `go get -u honnef.co/go/tools/cmd/staticcheck@latest`

Comment (akalinin-stripe) [reply .github/workflows/ci.yml:21]: Or `go install honnef.co/go/tools/cmd/staticcheck@latest`

Comment (akalinin-stripe) [reply .github/workflows/ci.yml:21]: Doesn't seem like this change got applied for some reason 🤷

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+          go install github.com/go-bindata/go-bindata/...@latest &&
```
Comment (richardm-stripe) [inline .github/workflows/ci.yml:22]:
```suggestion
          go install github.com/go-bindata/go-bindata/...@latest &&
          go install honnef.co/go/tools/cmd/staticcheck@latest &&
```

**Review by akalinin-stripe — APPROVED** [review]
Looks good!

**Pull request merged**